region/norfair/east.json

```diff
@@ -1470,115 +1709,197 @@
           "to": [
             {
               "id": 1,
-              "requires": [
-                {"or": [
-                  {"canComeInCharged":{
-                    "fromNode": 2,
-                    "framesRemaining": 30,
-                    "shinesparkFrames": 25
-                  }},
-                  {"canComeInCharged":{
-                    "fromNode": 3,
-                    "framesRemaining": 30,
-                    "shinesparkFrames": 35
-                  }},
-                  "canSpringBallWalljumpMidAir",
-                  {"and": [
+              "strats": [
+                {
+                  "name": "Base",
+                  "notable": false,
+                  "requires": [
+                    {"or":[
+                      "canFly",
+                      {"and": [
+                        "HiJump",
+                        "canSpringBallJumpMidAir"
+                      ]}
+                    ]}
+                  ]
+                },
+                {
+                  "name": "Bubble Mountain Springwall",
+                  "notable": true,
+                  "requires": ["canSpringBallWalljumpMidAir"]
+                },
+                {
+                  "name": "Bubble Mountain Delayed Walljump (Left)",
```

Comment:
its not a delayed walljump.

There is a delayed walljump strat not listed here, with no other requirements.  But its pretty precise.

Reply:
Indeed, it's doable without a delayed one.
Is the naked one comparable to writg?

Reply:
It feels a frame or two easier, but is comparable

Reply:
Ok, I put it behind the same `canInsaneWalljump` tech.